### ISSUE TITLE
The test results field is the wrong location

### ISSUE THREAD

**Issue by Sebastiaan127001** [ICTU/quality-time#9813]
**Describe the bug**
The input field named 'test results', that allows for filtering test results from the test results source, is placed at the test cases source. This is confusing for the end user.

**To Reproduce**
Steps to reproduce the behaviour:
1. Create a test cases metric
2. Add Jira as a source for test cases (with valid JQL)
3. Add JUnit as a source for test results
4. see the input filed named 'test results' at wrong source

**Expected behaviour**
Place the test results field at the test results source.

**Screenshots**
![image](https://github.com/user-attachments/assets/2edad600-8666-4ad9-89b7-ce4831d59c14)

**Comment by fniessink** [comment]
I think the location is correct. Otherwise, if you had multiple test reports, it would be possible to count different test results from different test reports and get a total count that's meaningless. For example, you could set the test results for report 1 to only passed tests and the test results for report 2 to only failed tests. The measurement would be the count of LTCs reported as passed in report 1 plus the count of LTCs reported as failed in report 2.

**Comment by Sebastiaan127001** [comment]
I agree with your explanation. Still I think placing the 'test results' filter at the Jira source configuration can be confusing to the end user. This because the complexity of this metric having two sources, a nominator (test results in junit) and a denominator (the total of planned tests in Jira). 

As a user I expect to configure the total of planned test cases by means of a Jira query in JQL.
To me, it is confusing that I configure the test results in the same visual pane.

We need to find a way to configure this 'test results' parameter for all the test result sources at once.
